frontend/src/views/FuelCodes/_schema.jsx

```diff
@@ -184,560 +205,3 @@ export const fuelCodeColDefs = (t) => [
     minWidth: 600
   }
 ]
-
-export const addEditSchema = {
```

Comment:
was this just left hanging around?

Reply:
Yes, this was unused when I finished the last ticket to combine add/edit/view into a single view but missed deleting this.